public/components/common/SignInControl.tsx

```diff
@@ -15,9 +15,15 @@ interface SignInControlProps {
 
 export const SignInControl: React.FunctionComponent<SignInControlProps> = (props) => {
   const fider = useFider()
+  const [showEmailForm, setShowEmailForm] = useState(fider.session.tenant.isEmailAuthAllowed)
```

Comment:
```suggestion
  const [showEmailForm, setShowEmailForm] = useState(fider.session.tenant ? fider.session.tenant.isEmailAuthAllowed : true)
```

this control is used for signup, so we'll need this.